src/main/webapp/app/tutor-course-dashboard/tutor-course-dashboard.component.ts

```diff
@@ -80,7 +87,9 @@ export class TutorCourseDashboardComponent implements OnInit {
                 this.numberOfSubmissions = this.stats.numberOfSubmissions;
                 this.numberOfAssessments = this.stats.numberOfAssessments;
                 this.numberOfComplaints = this.stats.numberOfComplaints;
+                this.numberOfOpenComplaints = this.stats.numberOfOpenComplaints;
```

Comment:
Ok, I only saw this now. So we already have this functionality in the `stats-for-tutor-dashboard` call, but you basically now do the same thing again for the `fort-tutor-dashboard` call. The question is then if we really need this code in both endpoints, or if one is sufficient. If we need it in both, then we should really refactor the server code into the `{Course,Complaints}Services`.
Furthermore, I looked into the `CourseResource` and we don't even populate the `numberOfOpenComplaints` and `numberOfOpenFeedbackRequests` fields in this case. We only do this in the `stats-for-instructor-dashboard` call (at least in the `CourseResource`. Please have a look at this process again and make it consistent.

Reply:
Ah, never mind my other comment then :D

Reply:
@ungaralex I agree with you, the server code needs to be refactored, and as we saw together on Tuesday, it is a bit more complex and out of the scope of this PR, as I didn't really add much to it. So is it ok for you if we refactor the server code for dashboard stats in a separate PR?

Reply:
Yes, as discussed I think it's fine if we create an issue for this and postpone this to a future PR